### PR TITLE
add initial structure

### PR DESCRIPTION
**Pull request by francisco25friday**

### CODE REVIEW

File: Dotnet/Friday.RockPaperScissors/.gitignore [hunk 1]
```diff
@@ -0,0 +1,298 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (vetras) [inline Dotnet/Friday.RockPaperScissors/.gitignore:1]: We could have only what we need here

File: Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors.Tests/RockPaperScissorsTest.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+using NUnit.Framework;
```
Comment (vetras) [inline Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors.Tests/RockPaperScissorsTest.cs:1]: i'm more fan of Xunit but ok

File: Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors.Tests/Friday.RockPaperScissors.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
```
Comment (vetras) [inline Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors.Tests/Friday.RockPaperScissors.Tests.csproj]: there are updates to these

File: Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors/Friday.RockPaperScissors.csproj [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
```
Comment (vetras) [inline Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors/Friday.RockPaperScissors.csproj:8]: cool

File: Dotnet/Friday.RockPaperScissors/.gitignore (hunk 1 above)
Comment (francisco25friday) [reply Dotnet/Friday.RockPaperScissors/.gitignore:1]: I just have gitignore file that I call the Super God GitIgnore.  That I copy-paste from .NET previous projects and don't need to worry about selecting which file to ignore. It just works :) But I don't mind trimming down the files only to the necessary ones.

**Pull request closed without merging**